Azure/azure-cli
# [Redis] Add support for 2024-03-01

**Related command**
az redis create
az redis update

**Description**
New api version 2024-03-01 that introduces argument to disable access keys authentication

**Testing Guide**

Deploy a Premium Azure Cache for Redis with Microsoft Entra Authentication configured and access keys disabled
`az redis create --location westus2 --name MyRedisCache --resource-group MyResourceGroup --sku Premium --vm-size p1 --disable-access-keys true --redis-configuration @"config_enable-aad.json"`
Disable access keys authentication for Redis.
`az redis update --name MyRedisCache --resource-group MyResourceGroup --set "disableAccessKeyAuthentication=true"`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
